src/modules/job-info/job_state.c

```diff
@@ -200,6 +230,8 @@ void job_state_destroy (void *data)
             }
             zlistx_destroy (&jsctx->futures);
         }
+        if (jsctx->early_annotations)
```

Comment:
Check for NULL not needed for any of the czmq destructors.

Reply:
was going to change, but i have the check for all of the other `zlistx` data structures in the same function.  I'll leave it for cleanup for another day.